tests/utils/LHNTestUtils.js

```diff
@@ -167,6 +170,63 @@ function getAdvancedFakeReport(isArchived, isUserCreatedPolicyRoom, hasAddWorksp
     };
 }
 
+/**
+ * @param {Number[]} [participantAccountIDs]
+ * @param {Number} [millisecondsInThePast] the number of milliseconds in the past for the last message timestamp (to order reports by most recent messages)
+ * @param {boolean} [isUnread]
+ * @returns {Object}
+ */
+function getFakeReportWithPolicy(participantAccountIDs = [1, 2], millisecondsInThePast = 0, isUnread = false) {
+    return {
+        ...getFakeReport(participantAccountIDs, millisecondsInThePast, isUnread),
+        type: CONST.REPORT.TYPE.CHAT,
+        chatType: CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT,
+        policyID: '08CE60F05A5D86E1',
+        oldPolicyName: '',
+        isOwnPolicyExpenseChat: false,
+        ownerAccountID: 1,
+    };
+}
+
+/**
+ * @param {Number} [id]
+ * @param {String} [name] the number of milliseconds in the past for the last message timestamp (to order reports by most recent messages)
```

Comment:
The doc for the param doesn't look correct.

Reply:
It definitely doesn't... I missed this.

Reply:
Sorry my bad, I copied and pasted the doc from another method.

Done [b9a9f8f](https://github.com/Expensify/App/pull/30068/commits/b9a9f8f19b20aa7f97ec0db0b85e24de395765b7)